.github/workflows/git-commit-checks.yml

```diff
@@ -0,0 +1,43 @@
+name: Git commit checker
+
+on:
+    pull_request:
+        # We don't need this to be run on all types of PR behavior
+        # See https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request
+        types:
+          - opened
+          - synchronize
+          - edited
```

Comment:
do we need reopened as well?

Reply:
I don't think so; reopened doesn't mean that the commits have changed.